lib/workers/repository/updates/flatten.ts

```diff
@@ -216,8 +217,21 @@ export async function flattenUpdates(
       update.semanticCommits = semanticCommits;
     }
   }
-  return updates
-    .filter((update) => update.enabled)
```

Comment:
This original code assumes that `enabled` is always boolean and never missing. My new code does not assume that.